web/src/scss/components/troubleshoot/SupportBundleList.scss

```diff
@@ -86,6 +86,16 @@
     }
   }
 }
+.deleting {
+  animation: blinker 3s linear infinite;
```

Comment:
adds blinking animation to the bundle that's being deleted

Reply:
this is neat